### ISSUE TITLE
League - Vanguard

### ISSUE THREAD

**Issue by onionmansz** [zhaodice/qemu-anti-detection#75]
Works after Vanguard?

**Comment by zhaodice** [comment]
nope, Vanguard service won't start, shows : 1: Incorrect function

**Comment by uragan1987** [comment]
@zhaodice can you give a video with your 1: Incorrect function

how you check it

**Comment by zhaodice** [comment]
> @zhaodice can you give a video with your 1: Incorrect function
> 
> how you check it

1.I installed the game which required my machine reboot
2.after reboot, the service won't start3
3.I just open the service manager , open the VGD service window
4.manually open/enable service it shows "1: Incorrect function"

![servlet](https://github.com/zhaodice/qemu-anti-detection/assets/63996691/e1e2a2d7-d007-44b3-b5b1-048462c725c2)
it just a example image from google(I uninstalled game for a long time)

**Comment by uragan1987** [comment]
Which game did you use?

**Comment by zhaodice** [comment]
> Which did you use?--Diese Nachricht wurde von meinem Android Mobiltelefon mit WEB.DE Mail gesendet.Am 14.05.24, 12:02 schrieb "赵怡然" ***@***.***>: @zhaodice can you give a video with your 1: Incorrect function how you check it I installed the game which required my machine reboot, after reboot, the service won't start,I just open the service manager , VGD —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

I use this anti-detection : https://github.com/zhaodice/proxmox-ve-anti-detection which like this project.

**Comment by uragan1987** [comment]
i cant see any application there for windows system
what is this for collection with application to check all antiVM methods?
can i download it some where? that looks great to check everythink

https://user-images.githubusercontent.com/63996691/185649897-b7609626-ee6d-42b1-bc5e-4465cb41a19a.png

**Comment by zhaodice** [comment]
> i cant see any application there for windows system what is this for collection with application to check all antiVM methods? can i download it some where? that looks great to check everythink
> 
> https://user-images.githubusercontent.com/63996691/185649897-b7609626-ee6d-42b1-bc5e-4465cb41a19a.png

https://github.com/zhaodice/qemu-anti-detection/issues/25
